tasks/Containerfile

```diff
@@ -48,10 +53,12 @@ RUN dnf -y update && \
         rpmdevtools \
         rsync \
         socat \
+        syslinux \
```

Comment:
This is certainly a curious thing to have in the tasks container -- certainly the rhel4edge builds from image builder already have a boot loader? We want to test these images, not apply a dozen hacks to fix their boot. What's that for?